app/controllers/application_templates_controller.rb

```diff
@@ -79,6 +79,28 @@ def toggle_unavailability_enabled
     redirect_back fallback_location: back_path
   end
 
+  def toggle_resume_upload_enabled
```

Comment:
Lotta repetition in code here. It's fine for now, but it would be worth looking at streamlining it for some later optimization.

Reply:
(like legit, keep it this way for now, it's too much for this PR)